robotbase/src/main/java/frc/robot/commands/coralRunner/CoralRunnerSetVelocity.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.commands.coralRunner;
+
+import edu.wpi.first.units.measure.AngularVelocity;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Robot;
+
+public class CoralRunnerSetVelocity extends Command {
+
+  private AngularVelocity m_velocity;
+  private boolean m_stopOnEnd;
+
+  public CoralRunnerSetVelocity(AngularVelocity velocity, boolean stopOnEnd) {
```

Comment:
stopOnEnd is something we have done in previous years, but it is generally bad practice to run a motor outside of a command. Let's not enable this capability until we have a use case for it.

An example use case would be setting the motor to a prime RPM, and then waiting for a condition before starting another command that sets it to a different RPM. Stopping the motor here would result in jerky/bad movement in this use case.